services/ext/api.go

```diff
@@ -1447,14 +1447,6 @@ func (api *PublicAPI) StorePubsubTopicKey(topic string, privKey string) error {
 	return api.service.messenger.StorePubsubTopicKey(topic, p)
 }
 
-func (api *PublicAPI) AddStorePeer(address string) (peer.ID, error) {
```

Comment:
Before removing this from API, please ensure with mobile team that they don't use this endpoint.
I checked the desktop, it is not used on our end.